containers/unit-tests/run.sh

```diff
@@ -32,7 +32,7 @@ case $ARCH in
 esac
 
 if [ -d bots ]; then
-    bots/test-bots
+    GITHUB_BASE=`git remote get-url origin` bots/test-bots
```

Comment:
The value of GITHUB_BASE is actually never used in the tests (and it shouldn't, since they are unit tests).  Setting it just allows the global "api" object in task/__init__.py to be constructed.  I think we should get rid of that global object, or at least construct it lazily in the medium term.

So what about `GITHUB_BASE=unused` and a comment like "We only set GITHUB_BASE so that "import task" works without failure."